servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder-arm/README.md

```diff
@@ -0,0 +1,186 @@
+
+```shell
+You have logged in. Now let us find all the subscriptions to which you have access...
+
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "home-Tenant-Id",
+    "id": "subscription-id",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "Subscription-Name",
+    "state": "Enabled",
+    "tenantId": "0envbwi39-TenantId",
+    "user": {
+      "name": "your-username@domain.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+If you have more than one subscription, specify the subscription-id you want to use with command below:
+```shell
+az account set --subscription <your-subscription-id>
+```
+
+### Provision the Resources
+
+After login Azure CLI with your account, now you can use the terraform script to create Azure Resources.
+
+#### Run with Bash
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=./terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=./terraform apply -auto-approve
+
+```
+
+#### Run with Powershell
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=terraform apply -auto-approve
+
+```
+
+It may take a few minutes to run the script. After successful running, you will see prompt information like below:
+
+```shell
+azurecaf_name.resource_group: Creating...
+azurecaf_name.azurecaf_name_servicebus: Creating...
+azurecaf_name.azurecaf_name_servicebus: Creation complete ...
+azurecaf_name.resource_group: Creation complete ...
+azuread_application.servicebusqueuebinder: Creating...
+azuread_application.servicebusqueuebinder: Creation complete ...
+azuread_application_password.servicebusqueuebinder: Creating...
+azuread_service_principal.servicebusqueuebinder: Creating...
+azuread_service_principal.servicebusqueuebinder: Creation complete ...
+azuread_application_password.servicebusqueuebinder: Creation complete ...
+azurerm_resource_group.main: Creating...
+azurerm_resource_group.main: Creation complete ...
+azurerm_servicebus_namespace.servicebus_namespace: Creating...
+...
+azurerm_servicebus_namespace.servicebus_namespace: Creation complete ...
+azurerm_role_assignment.role_servicebus_data_contributor: Creating...
+azurerm_servicebus_queue.queue: Creating...
+azurerm_servicebus_queue.queue: Creation complete ...
+...
+azurerm_role_assignment.role_servicebus_data_contributor: Creation complete ...
+
+Apply complete! Resources: 9 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+...
+```
+
+You can go to [Azure portal](https://ms.portal.azure.com/) in your web browser to check the resources you created.
+
+### Export Output to Your Local Environment
+Running the command below to export environment values:
+
+#### Run with Bash
+
+```shell
+source ./terraform/setup_env.sh
+```
+
+#### Run with Powershell
+
+```shell
+ . terraform\setup_env.ps1
+```
+
+## Run Locally
```

Comment:
Yonghui did some changes to this part, please also apply to this PR. https://github.com/Azure-Samples/azure-spring-boot-samples/pull/212